### DISCUSSION TITLE
New Gas Giant Landscape Pictures

### DISCUSSION THREAD

**Discussion by RisingLeaf** [endless-sky/endless-sky#8344] [Ideas]
Right now all gas giants feature landscapes like this one:

![nasa8](https://user-images.githubusercontent.com/85687254/218243663-09d36c3a-b06d-4bac-903b-97a8c64500f5.jpg)

While they look fine, they are pictures from far above the gas giant. When I land on one of these planets I expect more than just, well something I basically see from space.

That is why I want to propose a more cloudy look, as if you really were diving through the giant. 

Here are two examples of pictures that could be used, but as always I am open for other proposals: 

this picture was created with stable diffusion by SteelSirokos:
![gasgiant1](https://user-images.githubusercontent.com/85687254/218243769-c11ab1c6-8046-4e52-be88-9382b30c37a5.png)

I lately experimented with blender and volume materials, I am not quite there yet, but you get the idea:
![gasgiant2](https://user-images.githubusercontent.com/85687254/218243826-463c5d82-8924-4ab3-86a0-05389bc573af.png)
![gasgiant3](https://user-images.githubusercontent.com/85687254/218243828-7d1d93fe-641c-49d3-a9e3-56b7ce7f5008.png)
![gasgiant4](https://user-images.githubusercontent.com/85687254/218243830-5bd552b0-2e2e-4fff-adad-1e3727df12c4.png)

I am pinging @Zitchas because of Remnant and Gas-Giants are connected.
Also @Azure3141 because they seem to know a lot about these planets.

**Comment by RisingLeaf** [comment]
This would also allow for more "personalized" pictures for gas giants and not just jupiter/saturn/neptun/uranus.

For example something like this for a planet that has diamond rain:
![gasgiant_diamon_rain2](https://user-images.githubusercontent.com/85687254/218244388-ef5e44d0-feca-4034-909e-186d09d09b3a.png)

Edit:
Also with looking at my own species, I could actually place floating cities in these pictures.

**Comment by RisingLeaf** [comment]
If other want to experiment with volumetrics, here is a blend.
Be careful though, these things need a claculation power and may crash you pc if you crank up resolution:
[Models/gasgiant_landscapes_v1.blend](https://github.com/RisingLeaf/IncipiasAsstes/blob/main/Models/gasgiant_landscapes_v1.blend)

**Comment by Tangle10** [comment]
I'm just a random player, and quite possibly the only person with this problem, but ever since I looked at one of those "this image is supposed to simulate what it's like to have a stroke" pictures for too long AI images give me headaches. Feel free to ignore, though.

**Reply by RisingLeaf** [reply]
Ok, just to make sure, this one gives you a headache?

![](https://user-images.githubusercontent.com/85687254/218243769-c11ab1c6-8046-4e52-be88-9382b30c37a5.png)

cause the other ones are made with a 3D modelling software

**Reply by Tangle10** [reply]
> Ok, just to make sure, this one gives you a headache?
> 
> ![](https://user-images.githubusercontent.com/85687254/218243769-c11ab1c6-8046-4e52-be88-9382b30c37a5.png)
> 
> cause the other ones are made with a 3D modelling software

Yeah, the others are just fine for me. Thank you for checking in!

**Reply by RisingLeaf** [reply]
Interesting, thank you.
I do not expect us to put AI art in the game anytime soon due to copyright being unclear, so you do not have to worry
